contracts/test/base/registry/DelegationProxy.t.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.23;
+
+import "forge-std/Test.sol";
+import {ERC20Votes} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+import {LibClone} from "solady/utils/LibClone.sol";
+import {UpgradeableBeacon} from "solady/utils/UpgradeableBeacon.sol";
+import {Initializable_AlreadyInitialized} from "contracts/src/diamond/facets/initializable/Initializable.sol";
+import {DeployRiverBase} from "contracts/scripts/deployments/utils/DeployRiverBase.s.sol";
+import {DelegationProxy} from "contracts/src/base/registry/facets/distribution/v2/DelegationProxy.sol";
+
+contract DelegationProxyTest is Test {
+  DeployRiverBase internal deployRiverTokenBase = new DeployRiverBase();
+  address internal river;
+  address internal beacon;
```

Comment:
Should we move this to storage slot or keep inline?

Reply:
What do you mean? It's a test contract.

Reply:
wrong submission.